cliquet/authorization.py

```diff
@@ -39,6 +39,11 @@ class RouteFactory(object):
 
     def __init__(self, request):
         service = utils.current_service(request)
+
+        # Handle blank requests
+        if service is None:
+            return
```

Comment:
What makes it fails is when you create the configuration passing a root_factory in order use the RouteFactory everywhere.